README.md

```diff
@@ -122,6 +122,14 @@ Thanks to **[Tobias Nyholm](https://github.com/Nyholm)** and [Artem Zhuravlov](h
     // Each of methods above has last optional $user parameter
     // that allows to get/set per-user settings
     // Your User Entity must implement SettingsOwnerInterface if you wish to use per-user settings
+    // Example: class User extends BaseUser implements SettingsOwnerInterface
+    //
+    // You need to implement the getSettingIdentifier()
+    // Example:
+    // public function getSettingIdentifier()
+    // {
+    //     return $this->id;
```

Comment:
Sorry. I believe that you should have an identifier like: 
```php
return 'user_'.$this->id;
```

When prefixing with a string you avoid collisions if you have multiple entities in your application that could own a setting. 
```php
class Company implements SettingsOwnerInterface {
  public function getSettingIdentifier() {
    return 'cmpy_'.$this->id;
  }
}
```